docs/references/express/overview.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Clerk Express.js SDK
+description: Learn how to integrate Express.js into your Clerk application.
+---
+
+# Clerk Express.js SDK
+
+## Set up Clerk Express.js
+
+<Steps>
+
+### Create a Clerk application
+
+You need to create a Clerk application in your Clerk Dashboard before you can set up Clerk Express.js. For more information, check out our [Set up your application](/docs/quickstarts/setup-clerk) guide.
+
+### Install `@clerk/express`
+
+Once a Clerk application has been created, you can install and then start using Clerk Express.js in your application. An ESM module for the Clerk Express SDK is available under the [`@clerk/express`](https://www.npmjs.com/package/@clerk/express) npm package.
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+```sh filename="terminal"
+npm install @clerk/express
+```
+
+```sh filename="terminal"
+yarn add @clerk/express
+```
+
+```sh filename="terminal"
+pnpm add @clerk/express
+```
+</CodeBlockTabs>
+
+### Set environment variables
+
+The developers have to load the environment variables from the .env.local file by using dotenv or any other library of their preference.
+Below is an example of an .env.local file. 
+
+**Pro tip!** If you are signed into your Clerk Dashboard, your secret key should become visible by clicking on the eye icon. Otherwise, you can find your keys in the Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page.
+
+<InjectKeys>
+
+```env filename=".env.local"
+CLERK_PUBLISHABLE_KEY={{pub_key}}
+CLERK_SECRET_KEY={{secret}}
+```
+
+</InjectKeys>
+
+</Steps>
+
+## Available methods
+
+All resource operations are mounted as sub-APIs on the `clerkClient` object. To access the resource operations, [you must first instantiate a `clerkClient` instance](/docs/references/express/available-methods).
```

Comment:
❓ Do we need this section here? The `clerkClient` is also exported in the `@clerk/clerk-sdk-node`.